app/lib/esQueryBuilder.js

```diff
@@ -5,13 +5,73 @@ function build(searchTerm) {
     body: {
       size: 30,
       query: {
-        multi_match: {
-          query: searchTerm,
-          fields: ['name^2', 'address.addressLines', 'doctors'],
-          type: 'cross_fields',
-          operator: 'and'
+        bool: {
+          should: [
+            {
+              match_phrase: {
+                name: {
+                  query: searchTerm,
+                  boost: 2,
+                  slop: 1
+                }
+              }
+            },
+            {
+              match_phrase: {
+                alternativeName: {
+                  query: searchTerm,
+                  boost: 2,
+                  slop: 1
+                }
+              }
+            },
+            {
+              common: {
+                name: {
+                  query: searchTerm,
+                  cutoff_frequency: 0.0001
```

Comment:
The cutoff_frequency value is arbitrary - drawn from the examples from the ES documentation and which empirically seem to give good results based on the search terms in the analytics report.